src/BlazorUI/Bit.BlazorUI/Components/Layouts/Footer/BitFooter.scss

```
@@ -1,31 +1,192 @@
-﻿@import "../../../Styles/functions.scss";
+@import "../../../Styles/functions.scss";
```

Stylelint (17.14.0)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)